### PR TITLE
Ignore numpy 1.20 deperecation for pytest

### PR DESCRIPTION
**Pull request by take-cheeze**
Rel: https://github.com/pfnet/pytorch-pfn-extras/pull/111 and https://ci.preferred.jp/pytorch-pfn-extras.torch17-win/68241/

### CODE REVIEW

**Comment by take-cheeze** [comment]
Now ignite 0.4.3 dropped 3.5... https://github.com/pytorch/ignite/releases/tag/v0.4.3

**Comment by kmaehashi** [comment]
Ignite issue should be fixed in #113.

File: pytest.ini [hunk 1]
```diff
@@ -8,5 +8,7 @@ filterwarnings =
     ignore:Using or importing the ABCs from:DeprecationWarning:pyreadline
     # onnx calling deprecated API of google.protobuf
     ignore:Call to deprecated create function:DeprecationWarning:onnx
+    # numpy 1.20 deprecation for np.object referencing
+    ignore:.*is a deprecated alias for the builtin:DeprecationWarning
```
Comment (kmaehashi) [inline pytest.ini:12]: Let's make sure that deprecations are not coming from our code 😃 
```suggestion
    ignore:.*is a deprecated alias for the builtin:DeprecationWarning:onnx
```

**Comment by kmaehashi** [comment]
`tensorboard` has the same issue... Could you add a line for tensorboard as well? :pray:

**Comment by kmaehashi** [comment]
Thanks, LGTM!

**Pull request merged**